src/rpc/util.cpp

```diff
@@ -497,7 +498,7 @@ std::string RPCHelpMan::ToString() const
         if (i == 0) ret += "\nArguments:\n";
 
         // Push named argument name and description
-        sections.m_sections.emplace_back(std::to_string(i + 1) + ". " + arg.m_name, arg.ToDescriptionString());
+        sections.m_sections.emplace_back(::ToString(i + 1) + ". " + arg.m_name, arg.ToDescriptionString());
```

Comment:
Maybe I got it wrong but is the RPC interface locale independent anyways?

Reply:
The intention is that the RPC should be 100% locale independent :)

Reply:
Ok then never mind.